src/main/webapp/courseLoader.js

```diff
@@ -1,127 +1,200 @@
+    /* dummyPerceptionRating */ [11],
+    /* dummyPerceptionRating */ [5],
+  ].concat(tempTermPerceptionList);
+  const termPerceptionData = new google.visualization.arrayToDataTable(
+    termPerceptionList
+  );
+  const termPerceptionOptions = {
+    title: "Perception of Term Reviews",
+    legend: { position: "none" },
+    vAxis: { title: "Perception" },
+    hAxis: { title: "Comment Quantity" },
+    histogram: {
+      hideBucketItems: true,
+    },
+  };
+  const termPerceptionChart = new google.visualization.Histogram(
+    document.getElementById("term-chart")
+  );
+  termPerceptionChart.draw(termPerceptionData, termPerceptionOptions);
+
+  const tempProfPerceptionList = dataObject.termPerceptionList;
+  const profPerceptionList = [
+    ["Professor Perception"],
+    /* dummyPerceptionRating */ [2],
+    /* dummyPerceptionRating */ [9],
+  ].concat(tempProfPerceptionList);
+  const profPerceptionData = new google.visualization.arrayToDataTable(
+    profPerceptionList
+  );
+  const profPerceptionOptions = {
+    title: "Perception of Professor Reviews",
+    legend: { position: "none" },
+    vAxis: { title: "Perception" },
+    hAxis: { title: "Comment Quantity" },
+    histogram: {
+      hideBucketItems: true,
+    },
+  };
+  const profPerceptionChart = new google.visualization.Histogram(
+    document.getElementById("prof-chart")
+  );
+  profPerceptionChart.draw(profPerceptionData, profPerceptionOptions);
+}
 
 //TODO : Move params into request body
 
-function passData(){
-    const queryString = window.location.search;
-    const urlParams = new URLSearchParams(queryString);
-    const courseName = urlParams.get("course-name");
-    const term = urlParams.get("term");
-    const profName = urlParams.get("prof-name");
-    const units = urlParams.get("num-units")
-    const schoolName = urlParams.get("school-name")
-
-    const termInput = document.getElementById("term-input")[0].value;
-    const profInput = document.getElementById("prof-input")[0].value;
-    const ratingTerm = document.getElementById("rating-term").value;
-    const ratingProf = document.getElementById("rating-prof").value;
-    const hours = document.getElementById("hoursOfWork").value;
-    const diff = document.getElementById("difficulty").value;
-
-    const url = new URL("/data", window.location.origin);
-    url.searchParams.set("course-name", courseName);
-    url.searchParams.set("prof-name", profName);
-    url.searchParams.set("num-units", units);
-    url.searchParams.set("term", term);
-    url.searchParams.set("school-name", schoolName);
-    
-    url.searchParams.set("hourOfWork", hours);
-    url.searchParams.set("difficulty", diff);
-    url.searchParams.set("term-input", termInput);
-    url.searchParams.set("prof-input", profInput);
-    url.searchParams.set("rating-term", ratingTerm);
-    url.searchParams.set("rating-prof", ratingProf);
-    fetch(url, { method: "POST" });
-}
\ No newline at end of file
+function passData() {
```

Comment:
FYI this is going to conflict with Laura's PR, since she's doing the TODO.

Reply:
Yah I think we are both aware of this, whose ever PR gets submitted latest will have to sort the merge issues. Or honestly I can remove it from this PR because the main focus of this PR was the graph updates. Is there anything wrong with that approach?

Reply:
Nope, that sounds fine.